i18n-embed-fl is incompatible with the most recent fluent-bundle version 0.15.2

It seems  fluent-bundle version 0.15.2 introduced some breaking changes, so to use i18n-embed-fl I had to add
```toml
fluent-bundle = "=0.15.1"
```
to my Cargo.toml

With 0.15.2 version there are 5 errors:
```
error[E0277]: the trait bound `ExprAssign: Parse` is not satisfied
  --> /home/vlad/.cargo/registry/src/github.com-1ecc6299db9ec823/i18n-embed-fl-0.6.0/src/lib.rs:45:26
   |
45 |             if lookahead.parse::<syn::ExprAssign>().is_err() {
   |                          ^^^^^ the trait `Parse` is not implemented for `ExprAssign`

error[E0277]: the trait bound `ExprAssign: Parse` is not satisfied
  --> /home/vlad/.cargo/registry/src/github.com-1ecc6299db9ec823/i18n-embed-fl-0.6.0/src/lib.rs:52:40
   |
52 |             while let Ok(expr) = input.parse::<syn::ExprAssign>() {
   |                                        ^^^^^ the trait `Parse` is not implemented for `ExprAssign`

error[E0609]: no field `left` on type `ExprAssign`
  --> /home/vlad/.cargo/registry/src/github.com-1ecc6299db9ec823/i18n-embed-fl-0.6.0/src/lib.rs:53:60
   |
53 |                 let argument_name_ident_opt = match &*expr.left {
   |                                                            ^^^^ unknown field

error[E0609]: no field `left` on type `ExprAssign`
  --> /home/vlad/.cargo/registry/src/github.com-1ecc6299db9ec823/i18n-embed-fl-0.6.0/src/lib.rs:62:34
   |
62 | ...                   expr.left.span(),
   |                            ^^^^ unknown field

error[E0609]: no field `right` on type `ExprAssign`
  --> /home/vlad/.cargo/registry/src/github.com-1ecc6299db9ec823/i18n-embed-fl-0.6.0/src/lib.rs:73:43
   |
73 |                 let argument_value = expr.right;
   |                                           ^^^^^ unknown field
```

--- a/i18n-embed-fl/CHANGELOG.md
+++ b/i18n-embed-fl/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog for `i18n-embed-fl`
 
+## v0.6.1
+
++ Fix for #76, add missing `syn` dependency with `full` feature flag specified.
+
 ## v0.6.0
 
 ### Documentation

--- a/i18n-embed-fl/Cargo.toml
+++ b/i18n-embed-fl/Cargo.toml
@@ -2,7 +2,7 @@
 name = "i18n-embed-fl"
 description = "Macro to perform compile time checks when using the i18n-embed crate and the fluent localization system"
 categories = ["localization", "internationalization", "development-tools"]
-version = "0.6.0"
+version = "0.6.1"
 authors = ["Luke Frisken <l.frisken@gmail.com>"]
 edition = "2018"
 license = "MIT"
@@ -28,7 +28,7 @@ unic-langid = "0.9"
 [dependencies.syn]
 version = "1.0"
 default-features = false
-features = ["derive", "proc-macro", "parsing", "printing", "extra-traits"]
+features = ["derive", "proc-macro", "parsing", "printing", "extra-traits", "full"]
 
 [dev-dependencies]
 doc-comment = "0.3"

